docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -114,6 +114,139 @@ This will stop all processing and delete any internal topics in Kafka.
 The following changes in SQL syntax and functionality may mean SQL statements
 that ran previously no longer run.
 
+#### Any key name
```

Comment:
As above, or I can edit in a separate PR, whichever you prefer.

Reply:
I've done it.